canonical-web-and-design/jaas.ai
# Tidy up featured promos in /store

## Done

Tidy up featured promos in /store

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8029/store
- View new featured promos and verify they match the design in the ticket

## Details

Fixes #217
